contracts/options.cairo

```diff
@@ -185,6 +188,27 @@ func mint_option_token{syscall_ptr: felt*, pedersen_ptr: HashBuiltin*, range_che
                 strike_price=strike_price,
             );
         }
+
+        with_attr error_message("trade exceeds max liqpool deposit size"){
+            let (optsize_after) = IOptionToken.balanceOf(contract_address=option_token_address, account=user_address);
+            assert optsize_after.high = 0;
+            
+            let (optsize_diff: Uint256) = uint256_sub(optsize_after, optsize_before);
```

Comment:
omg... why are you calling balanceOf twice? you could have just used option_size... please use option_size instead of optsize_diff and remove the balanceOf calls